TicTacToe/TicTacToe/Models/GameLogicManager.swift

```diff
@@ -0,0 +1,76 @@
+import UIKit
+
+class GameLogicManager {
+    private let winningMoves = [[1,2,3], [4,5,6], [7,8,9],
```

Comment:
Good to add comment which describe the purpose of these types of variables/methods.